.github/workflows/prover-test.yaml

```diff
@@ -39,20 +39,6 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
-      - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
-      - name: install Prover dependencies
-        shell: bash
-        run: scripts/dev_setup.sh -b -p -y -J
-      - name: prepare move lang prover tooling.
-        shell: bash
-        run: |
-          echo 'Z3_EXE='/home/runner/bin/z3 | tee -a $GITHUB_ENV
-          echo 'CVC5_EXE='/home/runner/bin/cvc5 | tee -a $GITHUB_ENV
-          echo 'DOTNET_ROOT='/home/runner/.dotnet/ | tee -a $GITHUB_ENV
-          echo 'BOOGIE_EXE='/home/runner/.dotnet/tools/boogie | tee -a $GITHUB_ENV
-          echo 'MVP_TEST_ON_CI'='1' | tee -a $GITHUB_ENV
-          echo "/home/runner/bin" | tee -a $GITHUB_PATH
-          echo "/home/runner/.dotnet" | tee -a $GITHUB_PATH
-          echo "/home/runner/.dotnet/tools" | tee -a $GITHUB_PATH
+      - uses: aptos-labs/aptos-core/.github/actions/move-prover-setup@main
```

Comment:
Note that `@main` will not work until this PR is landed. You might want to use the relative path for the time being: `uses: ./.github/actions/move-prover-setup` after an initial `actions/checkout@v3` step

Reply:
done